src/test/java/com/rultor/agents/daemons/EndsDaemonITCase.java

```diff
@@ -114,46 +119,54 @@ public void readsExitCodeCorrectly() throws IOException {
     @Test
     @Ignore
     public void deprecatesDefaultImage() throws IOException {
-        final Talk talk = new Talk.InFile();
-        FileUtils.write(
-            new File(this.start(talk, "").getAbsolutePath(), "testing"), "12"
-        );
-        new EndsDaemon(new Profile.Fixed()).execute(talk);
-        for (final String path
-            : talk.read().xpath("/p/entry[@key='merge']/entry[@key='script']")
+        try (
+            final StartsDockerDaemon start =
+                new StartsDockerDaemon(Profile.EMPTY)
         ) {
-            if ("yegor256/rultor".equals(path)) {
-                final String dir = talk.read()
-                    .xpath("/talk/daemon/dir/text()").get(0);
-                MatcherAssert.assertThat(
-                    dir,
-                    StringContains.containsString(
-                        "#### Deprecation Notice ####"
+            final Talk talk = new Talk.InFile();
+            this.start(start, talk, "");
+            new EndsDaemon(new Profile.Fixed()).execute(talk);
+            for (final String path
+                : talk.read().xpath(
+                "/p/entry[@key='merge']/entry[@key='script']"
                     )
-                );
-                MatcherAssert.assertThat(
-                    dir,
-                    StringEndsWith.endsWith("##############")
-                );
+            ) {
+                if ("yegor256/rultor".equals(path)) {
+                    final String dir = talk.read()
+                        .xpath("/talk/daemon/dir/text()").get(0);
+                    MatcherAssert.assertThat(
+                        dir,
+                        StringContains.containsString(
+                            "#### Deprecation Notice ####"
+                        )
+                    );
+                    MatcherAssert.assertThat(
+                        dir,
+                        StringEndsWith.endsWith("##############")
+                    );
+                }
             }
         }
     }
 
     /**
      * Start a talk.
+     * @param start Docker daemon starter
      * @param talk Talk to start
      * @param stdout Std out
      * @return Home
      * @throws IOException In case of error.
      */
-    private File start(final Talk talk, final String stdout)
-        throws IOException {
-        Assume.assumeFalse(SystemUtils.IS_OS_WINDOWS);
-        final SSHD sshd = new SSHD(this.temp.newFolder());
+    private PfShell start(final StartsDockerDaemon start, final Talk talk,
+        final String stdout) throws IOException {
+        Assume.assumeTrue(
+            "true".equalsIgnoreCase(System.getProperty("run-docker-tests"))
```

Comment:
@mkordas subjectively I like `Assume.assumeTrue(new RunDockerTestsProperty.set())` since we don't hide the `Assume` better.
But either way both solutions seem to me are kind of falling into the same category as public static literals don't they. I mean even though we're instantiating classes here, in actuality the `assumeSet()` and `set()` methods are just `static` :(

=> I'd leave it as is and maybe remove the exclude anyhow (though see above usability comments ... need to think this through a little :) ) once Rultor is able to run these tests itself.